src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTests.cs

```diff
@@ -52,13 +52,13 @@ public static void Main()
             var model = compilation.GetSemanticModel(tree);
 
             var simple = tree.GetCompilationUnitRoot().DescendantNodes().OfType<SimpleLambdaExpressionSyntax>().Single();
-            Assert.True(((LambdaSymbol)model.GetSymbolInfo(simple).Symbol).IsAsync);
+            Assert.True(((IMethodSymbol)model.GetSymbolInfo(simple).Symbol).IsAsync);
```

Comment:
Consider additionally asserting that these are lambdas, as we're no longer casting them to `LambdaSymbol`s.

Reply:
> Consider additionally asserting that these are lambdas, as we're no longer casting them to LambdaSymbols.

I do not think we actually care about that fact.

---
In reply to: [339242045](https://github.com/dotnet/roslyn/pull/39498#discussion_r339242045) [](ancestors = 339242045)